test/rai/pipeline_adult_analyse.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
     inputs:
       training_data: ${{parent.inputs.my_training_data}}
       target_column_name: ${{parent.inputs.target_column_name}}
+      dropped_features: 'Race'
```

Comment:
should this be a list? '["Race"]'?

Reply:
It depends entirely on what we do inside the component. This is just a test component, not one we actually want people to use, so I don't know if it matters either way. I went with the human-readable comma-separated string for simplicity.